client_test.go

```diff
@@ -63,6 +65,46 @@ func defaultBenchmarkRotatorSettings(t *testing.B) *RotatorSettings {
 	return rotatorSettings
 }
 
+// sumRecordContentLengths returns the total Content-Length across all records in a WARC file.
```

Comment:
I'm not a huge fan of this, but I do think from a maintainability perspective, it is probably a good idea. 

What I was thinking: is that by having a specific byte range, we can validate that the WARC is "in spec" from when we last manually inspected it, but this does seem better 😅

Reply:
Added a separate test to keep doing it the "old way" too. Good!